AppCenter/AppCenter/Internals/Sender/MSHttpSender.m

```diff
@@ -351,6 +351,7 @@ - (NSURLSession *)session {
     NSURLSessionConfiguration *sessionConfiguration = [NSURLSessionConfiguration defaultSessionConfiguration];
     sessionConfiguration.timeoutIntervalForRequest = kRequestTimeout;
     _session = [NSURLSession sessionWithConfiguration:sessionConfiguration];
+    _session.delegateQueue.maxConcurrentOperationCount = 1;
```

Comment:
I'm sceptical here. We're limiting to one concurrent operation and we really shouldn't do that.